.gitignore

```
@@ -22,6 +22,7 @@ DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
+ios/.xcode.env.local
```

Can we anchor both this and the following `ios/` entries to the root `/ios` directory?
```suggestion
/ios/.xcode.env.local
```

---

Sure!

---

Regarding `gradle-wrapper.jar`: was in the diff helper to upgrade, changes are unknown.